v1.7/gke/PRODUCT.yaml

```diff
@@ -0,0 +1,6 @@
+vendor: Google
+name: Google Container Engine
+version: v1.7.6
+website_url: https://cloud.google.com/container-engine/
+documentation_url: https://cloud.google.com/container-engine/docs/
+product_logo_url: blah blah
```

Comment:
let's use https://cloud.google.com/_static/97fb761314/images/cloud/gcp-logo.svg same as the homepage